docs/src/tutorials/basics.md

```diff
@@ -73,6 +73,17 @@ savefig("basics_tree.png"); nothing # hide
 
 ![](basics_tree.png)
 
+To get the elementary differential, corresponding to a `RootedTree`, as a [`LaTeXString`](https://github.com/JuliaStrings/LaTeXStrings.jl), you can use [`elementary_differential`](@ref).
+
+```@example basics
+for t in RootedTreeIterator(4)
+    println(elementary_differential(t))
+end
+```
+In LaTeX this results in the following output:
+
+![]()
```

Comment:
To get the images you posted in your comment, you should be able to use something like

```suggestion
![f'f'f'f](https://user-images.githubusercontent.com/125130707/282897199-4967fe07-a370-4d64-b671-84f578a52391.png)
![f'f''ff](https://user-images.githubusercontent.com/125130707/282897199-4967fe07-a370-4d64-b671-84f578a52391.png)
![f''f'ff](https://user-images.githubusercontent.com/125130707/282897199-4967fe07-a370-4d64-b671-84f578a52391.png)
![f'''fff](https://user-images.githubusercontent.com/125130707/282897199-4967fe07-a370-4d64-b671-84f578a52391.png)
```

See for example https://github.com/trixi-framework/Trixi.jl/blob/7bb3b463a660edb5754fcf917ff8812f6791716a/docs/src/index.md?plain=1#L169C19-L169C112

Could you please build the docs locally to see whether everything is rendered correctly (or whether some blank lines are required)?